scripts/pysa_fuzzer/code_generator2.py

```diff
@@ -0,0 +1,1878 @@
+import os
```

Comment:
could we rename code_generator.py and code_generator2.py to be more descriptive?
For instance:
code_generator.py => forward_code_generator.py ("forward" because it generates code from top to bottom)
code_generator2.py => mutation_based_code_generator.py